packages/service/src/service.ts

```diff
@@ -10,8 +10,9 @@ import ThreeIdProvider from '3id-did-provider'
 import type { DIDMethodName, DIDProvider, DIDProviderMethods, DIDRequest, DIDResponse } from 'dids'
 import type { RPCErrorObject, RPCRequest, RPCResponse, RPCResultResponse } from 'rpc-utils'
 import Url from 'url-parse'
-import { UIProvider, ThreeIDManagerUI, AuthParams } from '../../../packages/ui-provider/src/index'
-import { expose } from 'postmsg-rpc'
+import { UIProvider, ThreeIDManagerUI, AuthParams } from '@3id/ui-provider'
+// import { expose } from 'postmsg-rpc'
+const { expose } = require('postmsg-rpc')
```

Comment:
just want to confirm this is intentional. I'm loathe to mix imports with requires.